deisctl/units/deis-controller.service

```diff
@@ -6,8 +6,10 @@ After=deis-store-volume.service
 [Service]
 EnvironmentFile=/etc/environment
 TimeoutStartSec=20m
+ExecStartPre=-/usr/bin/etcdctl mkdir /deis/scheduler >/dev/null 2>&1
 ExecStartPre=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/controller` && docker history $IMAGE >/dev/null 2>&1 || docker pull $IMAGE"
 ExecStartPre=/bin/sh -c "docker inspect deis-controller >/dev/null 2>&1 && docker rm -f deis-controller || true"
+ExecStartPre=-/bin/sh -c "etcdctl get /deis/scheduler/module >/dev/null 2>&1 || etcdctl mk /deis/scheduler/module 'fleet' "
```

Comment:
Using the `-` here means that this can fail. Do we want the unit to succeed if we can't set this key?